src/gpg/verify.py

```diff
@@ -0,0 +1,36 @@
+import sys
+import gpg
+import time
+
+
+def verify(signature_file, filename):
+    """
+    verify the signature
+    signature_file has the detached signature
+    filename has the name of the file which is signed
+    """
+    
+    c = gpg.Context()
+    
+    try:
+        _, result = c.verify(open(filename), open(signature_file))
+        verified = True
+    except gpg.errors.BadSignatures as e:
+        verified = False
+        print(e)
+
+    if verified is True:
+        for i in range(len(result.signatures)):
+            sign = result.signatures[i]
+            print(  
```

Comment:
Implement logging that shows logs to stdout and also saves to a file. 
Maybe implement it later by creating an issue?
 @dashohoxha  views?

Reply:
I have no idea why we would need logging for this.

On the other hand, there is an environment variable called `DEBUG`. If this is set, the program is expected to output more detailed information about what is going on.

I am not sure whether this can be implemented or whether it could be useful in this case.

Reply:
If you want I can use environment variable and maybe we can print the details of the objects here. Not sure if it is useful.

Reply:
>I have no idea why we would need logging for this.

Suppose an error have occurred and the person have closed the terminal or pressed page up or forgot the error, he/she may want to check the event that happened or one that leads to the error. Logging ensures that events are written to stdout and also to a file at the same time depending on our configuration(we can set levels respectively). 
It's better than redirection to a file as it have more configuration option.

Reply:
Now, that I think of it. It seems a bit overkill.

Reply:
> If you want I can use environment variable and maybe we can print the details of the objects here. Not sure if it is useful.

This is how DEBUG is used: https://github.com/EasyGnuPG/pgpg/blob/gnupg-2.2/src/platform.sh#L6
Maybe we can do something similar with GPGME.

Reply:
> Logging ensures that events are written to stdout and also to a file at the same time...

Does `gpg` have logging? Do CLI tools have logging in general? I think that logging is usually used in daemons or web applications.

Reply:
Yeah, I just thought of that. Seems stupid on my part.

Reply:
I found the following for this I think we can use this. https://www.gnupg.org/documentation/manuals/gpgme/Debugging.html#Debugging
i.e GPGME uses environment variable `GPGME_DEBUG` to point to a file for debug log. We can set the same in platform.sh.

Reply:
[mygpgme.log](https://github.com/EasyGnuPG/pgpg/files/2073486/mygpgme.log)

Reply:
Sample debug log for verify
[mygpgme.txt](https://github.com/EasyGnuPG/pgpg/files/2073489/mygpgme.txt)